test/tests/integration/backend.js

```diff
@@ -0,0 +1,70 @@
+import chai from "chai";
+const request = require("supertest");
+
+const { app, updateLumensCache } = require("../../../backend/app.js");
+describe("integration", function() {
+  // update caches
+  before(async function() {
+    await updateLumensCache();
+  });
+
+  describe("backend api endpoints", function() {
+    it("should return data of correct type", async function() {
```

Comment:
💡 I think these tests will be easier to maintain if there's a single test per endpoint. This test is a sort of mini-framework for a specific type of test, but it will probably be approximately the same effort to write them out one by one, and easier to modify then in each case. This is just my perspective though, and just a suggestion.

Example:
```js
   describe("backend api endpoints", function() {
     it("/api/lumens should return an empty object", async function() {
         let { body } = await request(app)
           .get("/api/lumens")
           .expect(200);
         chai.expect(body).to.be.an("object");
         chai.expect(Object.keys(data).length).to.eq(0);
     });
   });
 });
```

Reply:
agreed I like how they'll be easier to modify with your way 👍  changed
https://github.com/stellar/dashboard/pull/193/commits/7a9a3e5d4d21bc51729c179a8927a49fe43e0c30